alvr/client_openxr/src/interaction.rs

```diff
@@ -717,7 +717,11 @@ pub fn get_hand_data(
                         xr::SpaceLocationFlags::ORIENTATION_VALID
                             | xr::SpaceLocationFlags::POSITION_VALID,
                     ) {
-                        let time_offset_s = future_time.saturating_sub(time).as_secs_f32();
+                        let time_offset_s = future_time
+                            .saturating_sub(time)
+                            .max(Duration::from_millis(1))
+                            .as_secs_f32();
+
```

Comment:
I don't think trying to go back some safe val is the best idea, because that's still incorrect, just stop if `offset == 0` (or `<= 1ms`).
Also I recall seeing this and thinking it might be buggy, I just immediately forgor about it again.

Reply:
concur, maybe try https://doc.rust-lang.org/stable/std/primitive.i32.html#method.checked_div if it's available and then swap in an old value if the divide fails (if we can do that). The problem with epsilons and clamps for denominators is that if the time gets subtracted/added, it can accidentally kick the issue down the road to another divide

Reply:
Good call, i should replace all divisions with checked_div, the same way we have all Duration subtractions with saturating_sub